vendors/twtg/drivers/neon-vs-mt-xx-xx/package.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "neon-vs-mt-xx-xx",
```

```suggestion
  "name": "neon-vs-mt",
```

Same counts for the folder name

---

Corrected on MT Valve Sensor